### PR TITLE
CORTX-29541: Remove remaining references to openldap from README.md

### PR DESCRIPTION
**Pull request by walterlopatka**
This PR removes two references to openldap in README.md.  (These should have been removed as part of #141).

-----
[View rendered README.md](https://github.com/walterlopatka/cortx-k8s/blob/CORTX-29541_remove_openldap_from_readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Deploying and managing Kubernetes is outside the scope of this repository, howev
 ![CORTX on Kubernetes Reference Architecture](./doc/images/cortx-ref-arch-k8s.jpg)
 
 CORTX on Kubernetes consists of five primary components:
-1.  Prerequisite services, consisting of [Consul](https://github.com/hashicorp/consul), [Apache Kafka](https://kafka.apache.org/), and [OpenLDAP](https://www.openldap.org/).
+1.  Prerequisite services, consisting of [Consul](https://github.com/hashicorp/consul), [Apache Kafka](https://kafka.apache.org/).
```
Comment (osowski) [inline README.md]: I would just update this to use proper grammar.
```suggestion
1.  Prerequisite services, consisting of [Consul](https://github.com/hashicorp/consul) and [Apache Kafka](https://kafka.apache.org/).
```

**Review by osowski — CHANGES_REQUESTED** [review]
One minor grammar change.

**Comment by walterlopatka** [comment]
Note: This changeset shows three remaining instances of "ldap" in README.md.  These are being taken care of in #170.

File: README.md (hunk 1 above)
Comment (walterlopatka) [reply README.md]: I think I found it 30 seconds before you.  :)  Fixed.

**Review by osowski — APPROVED** [review]
lgtm

**Pull request merged**